exercises/practice/secret-handshake/.meta/example.rs

```diff
@@ -0,0 +1,18 @@
+const COMMANDS: [&str; 4] = ["wink", "double blink", "close your eyes", "jump"];
+
+pub fn commands(n: u8) -> Option<Vec<&'static str>> {
```

Comment:
I'm not sure about returning an `Option` value. I think returning an empty vector in the case of no commands would be fine. Option is usually only used when something can be absent, and I don't know if that is the case here.